steps/swift-package-manager-test-for-mac/0.0.1/step.yml

```diff
@@ -0,0 +1,54 @@
+title: Swift Package Manager Test for Mac
+summary: |
+  Runs Test on your macOS project with Swift Package Manager
+description: |
+  Runs Swift Package Manager's `swift test` action for macOS app projects.
+website: https://github.com/kitasuke/bitrise-step-swift-package-manager-test-for-mac
+source_code_url: https://github.com/kitasuke/bitrise-step-swift-package-manager-test-for-mac
+support_url: https://github.com/kitasuke/bitrise-step-swift-package-manager-test-for-mac/issues
+published_at: 2019-03-15T00:15:09.065797+09:00
+source:
+  git: https://github.com/kitasuke/bitrise-step-swift-package-manager-test-for-mac.git
+  commit: 07172e3cedac83dfd23f0fb5d435ae2ee4a78f88
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- macos
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/kitasuke/bitrise-step-swift-package-manager-test-for-mac
+deps:
+  brew:
+  - name: go
+  check_only:
+  - name: xcode
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_path: ./.build
+  opts:
+    description: |
+      A build path, relative to the working directory.
+    is_expand: true
+    is_required: false
```

Comment:
@kitasuke why is it not required? Can the step work without setting build_path?

Reply:
Can't for the step, but can work for `swift test` since default value is set for build_path. Do you think we should make it required for the case?

Reply:
I now feel like I should... Will fix it.

Reply:
Addressed